lib/search.rb

```diff
@@ -1098,8 +1098,20 @@ def self.ts_query(term: , ts_config:  nil, joiner: nil, weight_filter: nil)
     )
   end
 
+  def self.plain_to_tsquery(ts_config: nil, term:, joiner: nil)
+    ts_config = ActiveRecord::Base.connection.quote(ts_config) if ts_config
+    tsquery = "PLAINTO_TSQUERY(#{ts_config || default_ts_config}, #{ActiveRecord::Base.connection.quote(term)})"
+    tsquery = "REPLACE(#{tsquery}::text, '&', '#{PG::Connection.escape_string(joiner)}')::tsquery" if joiner
+    tsquery
+  end
+
+  private
```

Comment:
the private ... public... context switch is a bit confusing, can you move the private stuff to the end of the file?